packages/superflare/src/queue.ts

```diff
@@ -28,16 +28,20 @@ export async function handleQueue<Env>(
   config({ env, ctx });
 
   return await Promise.all(
-    batch.messages.map((message) => handleQueueMessage(message, ctx))
+    batch.messages.map((message) => handleQueueMessage(message, ctx, env))
   );
 }
 
-async function handleQueueMessage(message: Message, ctx: ExecutionContext) {
+async function handleQueueMessage<Env>(message: Message, ctx: ExecutionContext, env: Env) {
   const instance = await hydrateInstanceFromQueuePayload(
     message.body as MessagePayload
```

Comment:
I think it would make more sense to pass `ctx` and `env` here as second or third args, since that function is responsible for hydrating an instance. (even if we only end up assigning it for Job instances). Thoughts?